src/Serilog.Extensions.Hosting/Serilog.Extensions.Hosting.csproj

```diff
@@ -1,42 +1,56 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Description>Serilog support for .NET Core logging in hosted services</Description>
-    <VersionPrefix>4.2.1</VersionPrefix>
-    <Authors>Microsoft;Serilog Contributors</Authors>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
-    <LangVersion>8</LangVersion>
```

Comment:
If we remove the `LangVersion` restriction, we'll need a reference to `Nullable` on `netstandard2.0` to avoid dependencies on types that are missing on some underlying platforms, IIRC.

Reply:
I'm on it, I will push another commit for it.